activesupport/lib/active_support/core_ext/date_and_time/calculations.rb

```diff
@@ -205,13 +205,15 @@ def next_year
     # Week is assumed to start on +start_day+, default is
     # +Date.beginning_of_week+ or +config.beginning_of_week+ when set.
     # DateTime objects have their time set to 0:00 unless +same_time+ is true.
+    # +prev_week+ is also aliased as +last_week+.
```

Comment:
I think there is a way to rdoc find that. @zzak @fxn do you know how to make aliases being recognised?